tests/system-tests/rdscore/tests/00_validate_top_level.go

```diff
@@ -46,7 +46,7 @@ var _ = Describe(
 				rdscorecommon.VerifyGRMultipleConnectionIPv6ETPLocal)
 
 			It("Verify EgressService with Cluster ExternalTrafficPolicy",
-				Label("egress", "egress-etp-cluster"), reportxml.ID("76485"),
+				Label("egress", "egress-etp-cluster", "debug"), reportxml.ID("76485"),
```

Comment:
don't forget to remove it after debugging 😸

Reply:
sure